src/yalexs_ble/lock.py

```diff
@@ -479,6 +483,38 @@ def _parse_unix_timestamp(self, timestamp_bytes: bytes) -> datetime:
         _LOGGER.debug("%s: Parsed unix timestamp: %d", self.name, unix_timestamp)
         return datetime.fromtimestamp(unix_timestamp)
 
+    def _parse_operation_source(
+        self,
+        source: int,
+        remote_type: int,
+    ) -> tuple[LockOperationSource, LockOperationRemoteType | None]:
+        source_enum = VALUE_TO_LOCK_OPERATION_SOURCE.get(
+            source, LockOperationSource.UNKNOWN
+        )
+        remote_type_enum: LockOperationRemoteType | None = (
+            VALUE_TO_LOCK_OPERATION_REMOTE_TYPE.get(
+                remote_type, LockOperationRemoteType.UNKNOWN
+            )
+        )
+        if source_enum == LockOperationSource.UNKNOWN:
+            _LOGGER.info(
+                "%s: Unrecognized operation source code: %s",
+                self.name,
+                hex(source),
+            )
+
+        if source_enum == LockOperationSource.REMOTE:
```

Comment:
```suggestion
        if source_enum is LockOperationSource.UNKNOWN:
            _LOGGER.info(
                "%s: Unrecognized operation source code: %s",
                self.name,
                hex(source),
            )

        if source_enum is LockOperationSource.REMOTE:
```

enums are singletons so `is` should be used